tests/conftest.py

```diff
@@ -0,0 +1,19 @@
+"""Defines fixtures for tests."""
+
+import pytest
+
+
+@pytest.fixture()
```

Comment:
I claim we don't need the braces.
```suggestion
@pytest.fixture
```

Reply:
But that's an annoying enough comment that you should merge, and deal with it in the next py-hack-cookie-cut-athon.

Reply:
yeah I found that ruff fixed it for me when I removed the parentheses :/

according to ruff/flake8, parentheses 'on' is the default, but it can be turned off (see [here](https://pypi.org/project/ruff/0.0.221/#fixture-parentheses))